package.json

```diff
@@ -79,12 +79,12 @@
     "react-collapse": "^5.1.0",
     "react-content-loader": "^6.1.0",
     "react-dom": "^17.0.2",
```

Comment:
You'll notice I intentionally did not update to React 18. In order to do so, we must first enable the new architecture.